tests/trogdor/produce_spec_runner.py

```diff
@@ -212,11 +212,11 @@ def __init__(self, workload):
         value_payload_generator = get_payload_generator(self.value_generator_spec)
         self.key_generator = PayloadGenerator(key_payload_generator)
         self.val_generator = PayloadGenerator(value_payload_generator)
-        self.qps = self.produce_workload.get("targetMessagePerSec", 10000)
+        self.mps = self.produce_workload.get("targetMessagePerSec", 10000)
         self.max_messages = self.produce_workload.get("maxMessages", 100000)
         self.nr_finished_messages = 0
         self.nr_failed_messages = 0
         self.producer = create_producer_conn(self.bootstrap_servers, self.common_client_conf, self.producer_conf)
-        trogdor_log("Produce {} at message-per-sec {}".format(self.max_messages, self.qps))
+        trogdor_log("Produce {} at message-per-sec {}".format(self.max_messages, self.mps))
```

Comment:
This could be clearer, "Produce {} *messages* at .."

Reply:
Sure.